generator/feature_sorter.cpp

```diff
@@ -151,6 +151,8 @@ class FeaturesCollector2 : public FeaturesCollector
     GeometryHolder holder([this](int i) -> FileWriter & { return *m_geoFile[i]; },
                           [this](int i) -> FileWriter & { return *m_trgFile[i]; }, fb, m_header);
 
+    if (!fb.IsPoint())
```

Comment:
with the latest commit the most of this method's lines are changed anyway, so maybe its better to re-indent the block properly;
or 
```
if (fb.IsPoint())
  return;
```